q2_types/feature_data/tests/data/taxonomy/start_end_space_taxon.tsv

```diff
@@ -0,0 +1,2 @@
+Feature ID	Taxon	Confidence
+seq1	k__Foo; p__Bar 	-1.0
```

Comment:
Is this supposed to have leading and trailing whitespace? If so, I only see trailing whitespace here.

Reply:
Interesting, I went in and added a leading white space to the taxon column. I will push a new commit with the change.